src/entity/CommonGame.java

```diff
@@ -0,0 +1,108 @@
+package entity;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class CommonGame implements Game {
+    private final String ID;
+    private int score;
+    private final String difficulty;
+    private final String genre;
+    private final int initialLives;
+    private final int maxRounds;
+    private int currentLives;
+    private int roundsPlayed = 0;
+    private Round round;
```

Comment:
_Ties in with the comment about implementing `getCurrentRound()` not `getRound()`._ Ideally, we would track the rounds in a way where we don't have redundant data (i.e. spreading the same data across two different variables).

One way to approach this would be to track all of the rounds in a list and then assume that the last element in the list is the current round. Then, if we want to set a new current round we just add a round to the end of this list. This way there is only one source of truth and we don't have to worry about updating multiple variables with the same data.